reana_client/utils.py

```diff
@@ -207,32 +207,42 @@ def _validate_serial_workflow_environment(workflow_steps):
     :raises Warning: Warns user if the workflow environment is invalid in serial workflow steps.
     """
     for step in workflow_steps:
-        if ":" in step["environment"]:
-            environment = step["environment"].split(":", 1)
-            environment_image, environment_image_tag = environment[0], environment[-1]
-            if ":" in environment_image_tag:
-                logging.warning(
-                    "Image {} has invalid tag {}".format(
-                        environment_image, environment_image_tag
-                    )
-                )
-                sys.exit(1)
-            elif environment_image_tag in ENVIRONMENT_IMAGE_SUSPECTED_TAGS_VALIDATOR:
-                logging.warning(
-                    "Using '{}' tag is not recommended in {} image.".format(
-                        environment_image_tag, environment_image
-                    )
-                )
-                sys.exit(1)
-        else:
-            logging.warning(
-                "Image {} does not have an explicit tag.".format(step["environment"])
+        _validate_image_tag(step["environment"])
+
+
+def _validate_image_tag(image):
+    """Validate if image tag is valid."""
+
+    has_warnings = False
+    if ":" in image:
+        environment = image.split(":", 1)
+        image_name, image_tag = environment[0], environment[-1]
+        if ":" in image_tag:
+            click.secho(
+                "==> ERROR: Image {} has invalid tag '{}'".format(
```

Comment:
Shall we use everywhere in the output strings the expression "environment image", or even simply "environment", for better analogy with the `--environment` option name?